robotpy_build/tool.py

```diff
@@ -251,7 +251,7 @@ def add_subparser(cls, parent_parser, subparsers):
             help="Displays pyproject.toml after override processing",
             parents=[parent_parser],
         )
-        parser.add_argument("toml", nargs="?", default="pyproject.toml")
+        parser.add_argument("tomli", nargs="?", default="pyproject.toml")
```

Comment:
Please don't change this argument name. `tomli` as an argument name won't make much sense from a user perspective.